e2e/util/resources.js

```diff
@@ -537,6 +537,16 @@ exports.ACTIVITI7_APPS = {
             processwithvariables: "processwithvariables",
             simpleProcess: "simpleProcess",
             dropdownrestprocess: "dropdownrestprocess"
+        },
+        forms: {
```

Comment:
I don't see those forms in the version of the app at the moment present in GitHub

Reply:
Yes, i've added a new zip for simpleapp